wmn-data.json

```diff
@@ -159,7 +159,7 @@
         "name" : "adultism",
         "uri_check" : "https://www.adultism.com/profile/{account}",
         "e_code" : 200,
-        "e_string" : "static/r-1OqQ4o/css/www/main.css",
+        "e_string" : "profile-info",

```

Comment:
Can you change this to something that only appears once on the page with the account? Something like `last login:` or `Member since:`

Reply:
Sure. I changed it with commit 4daf7aba7d537a64003bfa54800fd152e993400c